docs/reference/logging/index.md

```diff
@@ -25,6 +25,16 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
+
+   --log-opt max-size=(unlimited)
+   --log-opt max-file=1
```

Comment:
This isn't rendered as code-block. Perhaps the indentation is 3 in stead of 4 spaces? You can test the docs by running `make docs`, which will build the and serve the docs in a container so that you can preview them in your webbrowser.